src/http/response.cr

```diff
@@ -54,7 +58,9 @@ class HTTP::Response
 
   def to_io(io)
     io << @version << " " << @status_code << " " << @status_message << "\r\n"
-    HTTP.serialize_headers_and_body(io, @headers, @body)
+    cookies = @cookies
+    headers = cookies ? cookies.add_request_headers(@headers) : @headers
```

Comment:
Yes, it should be add_response_headers, i'll fix that. It seems like i've missed serialization tests. That's why the tests works.